### PR TITLE
Update with notes from hybrid triage

### PR DESCRIPTION
**Pull request by garath**
Update the team dev guide with some additional points that have come up during Hybrid Triage meetings

### CODE REVIEW

**Comment by markwilkie** [comment]
Thanks @garath!

cc/ @dotnet/dnceng

**Review by markwilkie — DISMISSED** [review]

File: Documentation/TeamProcess/DevGuide/meetings.md [hunk 1]
```diff
@@ -12,13 +12,19 @@ In this new hybrid world there are many challenges ahead.  This is our current a
...
 - Every meeting is a Teams meeting.  (regardless if synchronous or async)
-- Meetings should start 10 minutes after the hour (or half hour) so that folks have breathing space in between.
+- Meetings should start after the hour (or half hour) so that folks have breathing space in between. At least five minutes are recommended, and ten minutes if reasonable.
 - Every meeting has recording turned on by default
 - A meeting agenda articulating what is hoped to be accomplished is highly encouraged.
 - Notes that articulate the main points discussed and conclusion are sent out after every meeting that is of general concern.  (1:1's don't count of course)
 - When folks are meeting physically together, laptops (or other devices) should be used to aid remote folks.  (see logistics note below)
 - The meeting organizer should *always* have a laptop.
+- Attendees (remote and in-person) should use the "hand raise" feature to request time to speak. The meeting organizer should monitor and use this list.
+- The meeting organizer should monitor Teams chat for relevent meeting information and especially funny memes.
```
Comment (MattGal) [inline Documentation/TeamProcess/DevGuide/meetings.md:22]: nit: relevant.  Also while it's fun to be funny I dunno if the memes are worth calling out here

**Review by MattGal — APPROVED** [review]

File: Documentation/TeamProcess/DevGuide/meetings.md [hunk 2]
```diff
@@ -12,13 +12,19 @@ In this new hybrid world there are many challenges ahead.  This is our current a
...
 - It's probably better to schedule meetings in a conference room and/or focus room so as to not disrupt the team room.  However, there are undoubtedly times when having the meeting in the team room is appropriate.
+- Only one person should speak at a time
+  - Modern conference room microphones are sensitive. Even quiet side conversations come through clearly to remote attendees.
```
Comment (adiaaida) [inline Documentation/TeamProcess/DevGuide/meetings.md:25]: Maybe add that any side conversations should be kept to Teams chats/the meeting chat

File: Documentation/TeamProcess/DevGuide/meetings.md (hunk 1 above)
Comment (garath) [reply Documentation/TeamProcess/DevGuide/meetings.md]: Fiiiiiiiiiinnnnnnnnnnnnnnnnnnnneeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee.

File: Documentation/TeamProcess/DevGuide/meetings.md [hunk 3]
```diff
@@ -12,13 +12,19 @@ In this new hybrid world there are many challenges ahead.  This is our current a
...
 ## Guidelines
 - Every meeting is a Teams meeting.  (regardless if synchronous or async)
-- Meetings should start 10 minutes after the hour (or half hour) so that folks have breathing space in between.
+- Meetings should start after the hour (or half hour) so that folks have breathing space in between. At least five minutes are recommended, and ten minutes if reasonable.
 - Every meeting has recording turned on by default
 - A meeting agenda articulating what is hoped to be accomplished is highly encouraged.
 - Notes that articulate the main points discussed and conclusion are sent out after every meeting that is of general concern.  (1:1's don't count of course)
 - When folks are meeting physically together, laptops (or other devices) should be used to aid remote folks.  (see logistics note below)
 - The meeting organizer should *always* have a laptop.
+- Attendees (remote and in-person) should use the "hand raise" feature to request time to speak. The meeting organizer should monitor and use this list.
```
[the remark this answers is not in the document]
Comment (garath) [reply Documentation/TeamProcess/DevGuide/meetings.md]: In a meeting the Meeting Organizer is omnipotent. I've added a note to make that explicit.

File: Documentation/TeamProcess/DevGuide/meetings.md (hunk 2 above)
Comment (garath) [reply Documentation/TeamProcess/DevGuide/meetings.md]: Added an explicit note along those lines.

File: Documentation/TeamProcess/DevGuide/meetings.md [hunk 4]
```diff
@@ -12,13 +12,20 @@ In this new hybrid world there are many challenges ahead.  This is our current a
...
 - Every meeting is a Teams meeting.  (regardless if synchronous or async)
-- Meetings should start 10 minutes after the hour (or half hour) so that folks have breathing space in between.
+- Meetings should start after the hour (or half hour) so that folks have breathing space in between. At least five minutes are recommended, and ten minutes if reasonable.
 - Every meeting has recording turned on by default
 - A meeting agenda articulating what is hoped to be accomplished is highly encouraged.
 - Notes that articulate the main points discussed and conclusion are sent out after every meeting that is of general concern.  (1:1's don't count of course)
 - When folks are meeting physically together, laptops (or other devices) should be used to aid remote folks.  (see logistics note below)
 - The meeting organizer should *always* have a laptop.
+- Attendees (remote and in-person) should use the "hand raise" feature to request time to speak. The meeting organizer, or her designee, should monitor and use this list.
+- The meeting organizer should monitor Teams chat for relevant meeting information.
```
Comment (adiaaida) [inline Documentation/TeamProcess/DevGuide/meetings.md:22]:
```suggestion
- The meeting organizer, or her designee, should monitor Teams chat for relevant meeting information.
```

**Review by adiaaida — APPROVED** [review]

**Review by adiaaida — APPROVED** [review]

**Pull request merged**